web/src/reactions.ts

```diff
@@ -58,6 +86,9 @@ function update_ui_and_send_reaction_ajax(message_id, rendering_details) {
     }
 
     const message = get_message(message_id);
+    if (message === undefined) {
+        return;
+    }
```

Comment:
What's your thinking on whether this should be an assert? I think probably this is impossible but can be cleaned up in a follow-up;  `process_reaction_click` could just get a `message` from its caller, and the callers of `reactions.toggle_emoji_reaction` also have a message object in the first place, so I think the right fix is to just have this function be passed a `Message` object in the first place.

Reply:
Ah yeah that would be way better, there was already fetching of the message up the stack (and handling of undefined messages) for all calls to this.